### ISSUE TITLE
User not able to record offsite polygon

### ISSUE THREAD

**Issue by ankitecd** [Plant-for-the-Planet-org/treemapper#467]
**Describe the bug**
User not able to record offsite polygon because of `Location not satisfied` error popping continuously

**Is it on a Production or Development Build?**
Production

**To Reproduce**
1. Go to Register trees > Multiple Trees > Offsite - Polygon
2. Select a location and continue
3. See error

**Screenshots**
![image](https://user-images.githubusercontent.com/25547888/130067374-7fc2646a-9e32-4196-a74e-85851f87e9cb.png)

**Smartphone (please complete the following information):**
- Device: OnePlus 8T
- OS: Android 11
- TreeMapper App Version: 1.0.5

**Additional context**
Problem was reported by Kim, part of our Salesforce partnership while trying to record Multiple Offsite - polygon.

**Comment by norbertschuler** [comment]
@ankitecd I can not reproduce this on my iPhone X - so maybe it's just an Android issue.

**Comment by norbertschuler** [comment]
@ankitecd I also do not get this error testing on my Samsung Galaxy S6 Edge. But when I remove the location permission or just do not grant it, I get exactly the above error screen. Maybe he just has rejected the location permission when asked for it?

**Comment by norbertschuler** [comment]
The iOS app is asking the user to open the settings to grant location permission. Is that also possible to do for Android?

![IMG_33C2CE42C946-1](https://user-images.githubusercontent.com/1532418/130070663-0ea60109-2c9c-4964-a6a0-d6049c5cfa38.jpeg)

**Comment by norbertschuler** [comment]
Why is this dialog offer opening the settings for iOS while retrying to get the location for Android at https://github.com/Plant-for-the-Planet-org/treemapper/blob/61603fb462422f8aba70f1dfcdf793e4d99eee2d/app/components/Common/MapMarking/index.js#L721-L741

I wonder why this was implemented differently at https://github.com/Plant-for-the-Planet-org/treemapper/blob/61603fb462422f8aba70f1dfcdf793e4d99eee2d/app/components/DownloadMap/index.js#L345-L363 or at https://github.com/Plant-for-the-Planet-org/treemapper/blob/61603fb462422f8aba70f1dfcdf793e4d99eee2d/app/components/RegisterSingleTree/index.js#L218-L234 or at https://github.com/Plant-for-the-Planet-org/treemapper/blob/61603fb462422f8aba70f1dfcdf793e4d99eee2d/app/components/TreeInventory/index.tsx#L393-L411

Maybe check all source if they are handling it correctly?

**Comment by ankitecd** [comment]
> Maybe check all source if they are handling it correctly?

Yes. Also I think we don't need to ask for location for offsite registrations, so probably I will remove those

**Comment by ankitecd** [comment]
> 
> 
> @ankitecd I also do not get this error testing on my Samsung Galaxy S6 Edge. But when I remove the location permission or just do not grant it, I get exactly the above error screen. Maybe he just has rejected the location permission when asked for it?

We were on meet where he also showed his screen. Location was on also on high accuracy but this error was popping again and he wasn't able to proceed ahead. Works on my device (Samsung M31s) as well but not on his.
